instfiles/km-00010416.toml

```diff
@@ -0,0 +1,1047 @@
+Key71=65474:0
+Key72=65475:0
+Key73=65476:0
+Key74=65477:0
+Key75=65478:0
+Key76=65479:0
+Key77=65407:0
+Key78=65300:0
+Key79=65429:0
+Key80=65431:0
+Key81=65434:0
+Key82=65453:45
+Key83=65430:0
+Key84=65437:0
+Key85=65432:0
+Key86=65451:43
+Key87=65436:0
+Key88=65433:0
+Key89=65435:0
+Key90=65438:0
+Key91=65439:0
+Key92=65377:0
+Key93=0:0
+Key94=124:124
+Key95=65480:0
+Key96=65481:0
+Key97=65360:0
+Key98=65362:0
+Key99=65365:0
+Key100=65361:0
+Key101=0:0
+Key102=65363:0
+Key103=65367:0
+Key104=65364:0
+Key105=65366:0
+Key106=65379:0
+Key107=65535:127
+Key108=65421:13
+Key109=65508:0
+Key110=65299:0
+Key111=65377:0
+Key112=65455:47
+Key113=65027:0
+Key114=269025049:0
+Key115=65515:0
+Key116=65516:0
+Key117=0:0
+Key118=269025153:0
+Key119=269025093:0
+Key120=269025094:0
+Key121=269025095:0
+Key122=269025096:0
+Key123=0:0
+Key124=65027:0
+Key125=65513:0
+Key126=65469:61
+Key127=65515:0
+Key128=65517:0
+Key129=0:0
+Key130=0:0
+Key131=0:0
+Key132=0:0
+Key133=0:0
+Key134=0:0
+Key135=0:0
+Key136=0:0
+
+[shiftcapslockaltgr]
+Key8=65041:0
+Key9=65307:27
+Key10=161:161
+Key11=189:189
+Key12=190:190
+Key13=188:188
+Key14=2756:8540
+Key15=168:168
+Key16=2758:8542
+Key17=2761:8482
+Key18=177:177
+Key19=176:176
+Key20=191:191
+Key21=65116:731
+Key22=65288:8
+Key23=65056:0
+Key24=47:47
+Key25=63:63
+Key26=176:176
+Key27=174:174
+Key28=956:359
+Key29=165:165
+Key30=2300:8593
+Key31=697:305
+Key32=248:248
+Key33=254:254
+Key34=96:96
+Key35=65108:175
+Key36=65293:13
+Key37=65507:0
+Key38=230:230
+Key39=223:223
+Key40=240:240
+Key41=170:170
+Key42=959:331
+Key43=689:295
+Key44=65122:0
+Key45=38:38
+Key46=435:322
+Key47=65113:733
+Key48=94:94
+Key49=172:172
+Key50=65505:0
+Key51=186:186
+Key52=60:60
+Key53=62:62
+Key54=169:169
+Key55=2813:8218
+Key56=2768:8216
+Key57=2769:8217
+Key58=924:0
+Key59=215:215
+Key60=247:247
+Key61=65110:729
+Key62=65506:0
+Key63=65450:42
+Key64=65511:0
+Key65=32:32
+Key66=65509:0
+Key67=65470:0
+Key68=65471:0
+Key69=65472:0
+Key70=65473:0
+Key71=65474:0
+Key72=65475:0
+Key73=65476:0
+Key74=65477:0
+Key75=65478:0
+Key76=65479:0
+Key77=65407:0
+Key78=65300:0
+Key79=65429:0
+Key80=65431:0
+Key81=65434:0
+Key82=65453:45
+Key83=65430:0
+Key84=65437:0
+Key85=65432:0
+Key86=65451:43
+Key87=65436:0
+Key88=65433:0
+Key89=65435:0
+Key90=65438:0
+Key91=65439:0
+Key92=65377:0
+Key93=0:0
+Key94=65109:728
+Key95=65480:0
+Key96=65481:0
+Key97=65360:0
+Key98=65362:0
+Key99=65365:0
+Key100=65361:0
+Key101=0:0
+Key102=65363:0
+Key103=65367:0
+Key104=65364:0
+Key105=65366:0
+Key106=65379:0
+Key107=65535:127
+Key108=65421:13
+Key109=65508:0
+Key110=65299:0
+Key111=65377:0
+Key112=65455:47
+Key113=65027:0
+Key114=269025049:0
+Key115=65515:0
+Key116=65516:0
+Key117=0:0
+Key118=269025153:0
+Key119=269025093:0
+Key120=269025094:0
+Key121=269025095:0
+Key122=269025096:0
+Key123=0:0
+Key124=65027:0
+Key125=65513:0
+Key126=65469:61
+Key127=65515:0
+Key128=65517:0
+Key129=0:0
+Key130=0:0
+Key131=0:0
+Key132=0:0
+Key133=0:0
+Key134=0:0
+Key135=0:0
+Key136=0:0
```

Comment:
This isn't a toml file - it's an ini file.

We use the ini format for v0.10.x and earlier. toml is used for devel. What is being targetted here?